src/types.ts

```diff
@@ -73,41 +74,3 @@ export interface Vulnerability {
   exploitTypes: string[];
   exploitUris: string[];
 }
-
```

Comment:
They even used the same type, endpoint had 'name' and Odatainformation as additional. The name was also on the machine response 🐛